x-pack/legacy/plugins/siem/public/components/inspect/modal.tsx

```diff
@@ -16,11 +18,25 @@ import {
   EuiSpacer,
   EuiTabbedContent,
 } from '@elastic/eui';
-import React from 'react';
+import numeral from '@elastic/numeral';
+import { getOr } from 'lodash/fp';
+import React, { ReactNode } from 'react';
 import styled from 'styled-components';
 
 import * as i18n from './translations';
 
+const DescriptionListStyled = styled(EuiDescriptionList)`
+  ${({ theme }) => `
+    dt {
+      font-size: ${getOr('12px', 'eui.euiFontSizeXS', theme)} !important;
```

Comment:
I would say that the reduction in `font-size` here might not be desirable, as the description list is laid out in columns, and this causes the title to be oddly sized compared to the description that appears to its right. I would go with the default EUI font size for the component.

Reply:
ok sounds good but I was just trying to re-apply what you did for every DescriptionList in the app

Reply:
You mean the ones on the host and IP details pages? Yeah, those description lists are a bit different layout-wise (vertical instead of horizontal). In this case, I think it makes sense for them to be the same font size. Thanks!